source/includes/azure/_instances.md

```diff
@@ -0,0 +1,269 @@
+  "imageOffer": "CoreOS",
+  "imageSku": "Stable",
+  "imageVersion": "1010.5.0",
+  "region": "canadacentral",
+  "networkSubnet": "10.0.0.0/22",
+  "username": "johndoe",
+  "password": "SomePassw0rdVal!d"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances</code>
+
+Create a new instance
+
+Required | &nbsp;
+------- | -----------
+`name` <br/>*string* | The name of the instance. The name cannot exceed 64 characters
+`machineType`<br/>*string* | The type of machine assigned for this instance
+`imagePublisher`<br/>*string* | The image publisher from which the image is selected
+`imageOffer`<br/>*string* | The image offer that was used to create the instance
+`imageSku`<br/>*string* | The image SKU that was used to create the instance
+`imageVersion`<br/>*string* | The image version that was used to create the instance
+`region`<br/>*string* | The region in which the instance is located
+`networkSubnet`<br/>*string* | The subnet that the instance will be part of
+`username`<br/>*string* | The administrator username which will be created on the instance. It cannot be a reserve user such as admin,root or administrator and must not be more than 20 characters.
+
+Optional | &nbsp;
+------- | -----------
+`password`<br/>*string* | The password of the administrator account. It must be between between 12 and 72 characters and must be a combination of 3 of the following patterns : Special lettes, Upper Case, Lower Case and Numbers. The password is mandatory if the sshkey is not provided.
+`sshkey`<br/>*string* | The ssh key public portion that will be assigned to the user on the machine. This cannot be used for a Windows based OS.
+
+<!-------------------- DELETE AN INSTANCE -------------------->
+
+#### Delete an instance
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/azure/example/instances/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Compute/virtualMachines/example-small-server"
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id</code>
+
+Delete an existing instance
+
+<!-------------------- CHANGE MACHINE TYPE -------------------->
+
+#### Change machine type
+
+A machine type determines the number of vCPUs and the size of the memory allocated to the [instances](#azure-instances).
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure/example/instances/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Compute/virtualMachines/example-small-server?operation=resize"
+
+# Request example:
+```
+
+```json
+{
+  "region" : "canadacentral",
+  "machineType": "Standard_B1ls"
```

Comment:
Can the region be changed?

Reply:
No. There is an item opened to correct that in the backlog

Reply:
Got it